fuzzers/074-dump_all/jobtiles.tcl

```diff
@@ -13,6 +13,27 @@ set root_fp [open "root_${blocknb}.csv" w]
 #puts $root_fp "filetype,subtype,filename"
 
 set tiles [get_tiles]
+set MAGIC 0.6875
```

Comment:
can you add some comment why do we need this?

Reply:
Done